tensorflow_addons/image/BUILD

```diff
@@ -1,25 +1,28 @@
 licenses(["notice"])  # Apache 2.0
```

Comment:
This is not how the `BUILD` is formatted. Please look at [Development tips](https://github.com/tensorflow/addons/blob/master/CONTRIBUTING.md#development-tips) to know how to auto-format your code. 

If you're on Windows, you can use docker to do the same.